### PR TITLE
Adding papers to the repo and links to them to the website.

### PR DESCRIPTION
**Pull request by daphnei**

### CODE REVIEW

**Comment by daphnei** [comment]
Once this PR goes through, we should email authors asking them to check that that all of the information for their paper is correct, and to email us if they see any issues.

File: index.md [hunk 1]
```diff
@@ -64,7 +64,113 @@ Some of the below sessions will occur on our Discord server and some will occur
...
+29. [Sculpting with Words](https://neuripscreativityworkshop.github.io/2021/accepted/srw_98.pdf)
+    * *Joel Simon, Victor Perez Muñoz, Tal Shiri*
+
+30. [Neural Design Space Exploration](https://neuripscreativityworkshop.github.io/2021/accepted/srw_99.pdf)
+    * *Wei Jiang, Richard Davis, Kevin Gonyop Kim, Pierre Dillenbourg*
+
+31. [Inspiration through Observation: Demonstrating the Influence of Generated Text on Creative Writing](https://neuripscreativityworkshop.github.io/2021/accepted/srw_102.pdf)
+    * *Melissa Roemmele*
+
+32. [Gaudí: Conversational Interactions with Deep Representations to Generate Image Collections](https://neuripscreativityworkshop.github.io/2021/accepted/srw_104.pdf)
```
Comment (tesfaldet) [inline index.md:160]: `srw` should be replaced with `ncw` no?

**Review by tesfaldet — CHANGES_REQUESTED** [review]
The paper links in `index.md` should be `ncw_#.pdf` and not `srw_#.pdf`.

File: index.md (hunk 1 above)
Comment (daphnei) [reply index.md]: Oops, mixed up the acronyms for the two workshops I am organizing.

**Review by tesfaldet — APPROVED** [review]
lgtm 👍

**Pull request merged**